MetaMask/metamask-mobile
# fix: incorrect confirmed transaction notification

## **Description**
Whenever I send a transaction that fails, I can see how the popup with Transaction Confirmed message appears on the screen.


## **Manual testing steps**

_1. Step1:_ Go to test dapp
_2. Step2:_ Deploy an ERC721 token
_3. Step3:_ Mint token
_4. Step4:_ Click Transfer
_5. Step5:_ Accept
_6. Step6:_ Before tx is confirmed Click Approve
_7. Step7:_ Accept Tx
_8. Step8:_ See how Confirmed Transaction popup is displayed
_9. Step9:_ Check on Etherscan
_10. Step10:_ See how transaction failed

Bitrise: https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/e32dccac-1823-4b94-9f4b-7917f08a0cde

## **Screenshots/Recordings**

### **Before**

https://github.com/MetaMask/metamask-mobile/assets/29962968/62c4d491-a95e-493c-95ab-9d77f2f63978



### **After**
https://recordit.co/8dMu4U47sB


## **Related issues**

_Fixes #6834 

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
